controllers/courses.js

```diff
@@ -127,12 +128,12 @@ const editCourseHandler = (req, res, next) => {
 		action += `?redirectUrl=${req.query.redirectUrl}`;
 	}
 
-	let classesPromise;
+	let classesAndGroupsPromise;
```

Comment:
For it would be mit simpler and more understandable if we would have two different promises and you use the one or the other if the feature flag is set or not set

Reply:
fixed